tests/roles/barbican_adoption/tasks/main.yaml

```diff
@@ -69,3 +69,17 @@
   until: barbican_responding_result is success
   retries: 60
   delay: 2
+
+- name: check that Barbican secret payload was migrated successfully
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+
+    alias openstack="oc exec -t openstackclient -- openstack"
+
+    secret_id=`${BASH_ALIASES[openstack]} secret list | grep testSecret | cut -d'|' -f 2 | xargs`
+      secret_payload=`${BASH_ALIASES[openstack]} secret get $secret_id --payload -f value`
```

Comment:
why the different in the indentation here (shouldn't matter, but, why?)

Reply:
(i can merge once i hear back just waiting for sanity check on this )

Reply:
My mistake, let me fix it.

Reply:
ack thanks ... as just discussed with @xek on slack, since this ran last night if you update just this bit we don't have to wait for a new ci run, I will override